src/main/java/teammates/common/datatransfer/FeedbackSessionResultsBundle.java

```diff
@@ -1709,7 +1709,11 @@ public String getQuestionText(String feedbackQuestionId) {
         return sortedMap;
     }
 
-    public LinkedHashMap<String, Map<FeedbackQuestionAttributes, List<FeedbackResponseAttributes>>>
+    /**
+     * Returns an ordered Map with {@code recipientTeam} name as key
+     * sorted by {@code compareByTeamQuestionRecipientTeamGiver} comparator.
```

Comment:
Agree. However, the sorting logic is a little bit complicated and it is not easy to describe it briefly. I was expecting the developers to look at the comparator directly but the comparator is private. Maybe we could give a high-level description like the comment for the sorter `// Sorts by recipientTeam > question > recipientName > giverTeam > giverName`.